packages/apollo-forest-run/src/values/execute.ts

```diff
@@ -300,14 +308,16 @@ function executeCompositeListSelection<TModel, TListModel>(
 
   let incompleteItems: Set<number> | undefined;
   if (draft) {
-    incompleteItems = incompleteValues.get(draft);
+    incompleteItems = incompleteValues.get(draft) as Set<number> | undefined;
```

Comment:
nitpick: i guess you do not need to overcast it. It already returns the same type.